autotest.pm

```diff
@@ -103,6 +103,12 @@ sub loadtest {
     }
     my $category = $1;
     my $name     = $2;
+    if ($category ne 'other') {
```

Comment:
Could you add a comment on top explaining the reason? Maybe even worth to pull out a function with a descriptive name

Reply:
A separate function would it also make a < 10 minute task to create a unit test for it.

Reply:
done.